bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/NamedThreadFactory.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
As this class is now merely used within the tests, I would say it should be removed (the tests do not really need it).

Reply:
It's removed.